contract_execution_argument.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @param {string|Object} argument
+ * @param {string} nonce
+ * @param {string[]} functionIds
+ * @return {string}
+ */
+function format(argument, nonce, functionIds) {
+  if (typeof argument !== 'string' && typeof argument !== 'object') {
+    throw new Error('argument must be a string or an object');
+  }
+
+  if (typeof nonce !== 'string') {
+    throw new Error('nonce must be a string');
+  }
+
+  if (!Array.isArray(functionIds)) {
+    throw new Error('functionIds must be an array');
+  }
+
+  return `V2\u0001${nonce}\u0003${functionIds
+      .filter((v) => typeof v === 'string')
+      .join('\u0002')}\u0003${
```

Comment:
Can we use some constants for these magic separators and the version string?

Reply:
@jnmt 
Thank you for the reminder.
I should have done this earlier.
I fixed this in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/81/commits/7ee78b41cfaaa5cd22b199d18f70488dd9fbd103
Please take a look.
Thank you!